book/chapters/methods.md

```diff
@@ -0,0 +1,15 @@
+# Machine Learning Methods and Tools
+
+Explain the method and why you think it's suitable for your use case.
+Explain the choice of tools/packages/data and the reason for use.
```

Comment:
Assuming that we just need to mention the machine learning algorithm here? Or do we need to include other tools/libraries as well (e.g. icepyx) that will be used in this Jupyter Book?

Reply:
I think icepyx, etc. would better fit into the 'data preparation' chapter.